src/game_interpreter_battle.cpp

```diff
@@ -91,6 +91,10 @@ bool Game_Interpreter_Battle::CommandCallCommonEvent(RPG::EventCommand const& co
 		return true;
 	}
 
+	if ((int)_state.stack.size() > call_stack_limit) {
+		Output::Error("Call Event limit has been exceeded");
```

Comment:
Before the message was: ``Interpreter: Maximum callstack depth (100) exceeded``, the 100 could be mentioned somewhere.

Reply:
Fixed